catalog/acm/config-mgmt-csr.yaml

```diff
@@ -0,0 +1,50 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+apiVersion: gkehub.cnrm.cloud.google.com/v1beta1
+kind: GKEHubFeatureMembership
+metadata:
+  name: feature-membership-name # kpt-set: acm-mem-${cluster-name}
```

Comment:
The name value seems out of sync with the interpolated setter. Also would `acm-mem` prefix seems odd since mem can also mean memory in a k8s manifest. Should we just go with `acm-membership`?

Reply:
I will change it to membership. I was trying to keep the name short. I have removed the value in the setter since it's derived from cluster name. This also reduces the amount of setter.

Reply:
changed the name